docker/py3-native/Dockerfile

```diff
@@ -0,0 +1,103 @@
+    && ldconfig \
+    && cd .. \
+    && git clone https://github.com/zdenop/tessdata_downloader.git \
+    && for lang in eng ara chi_sim chi_sim_vert chi_tra chi_tra_vert deu fra heb ita jpn jpn_vert pol por rus spa swe tur ; \
+    do python tessdata_downloader/tessdata_downloader.py -o /usr/local/share/tessdata/ -l $lang; done \
+    && tesseract --version --list-langs \
+    && rm -rf tessdata_downloader leptonica tesseract download_chromedriver.sh repos \
+    && dnf remove -y python3-devel gcc gcc-c++ make platform-python-pip unzip git make autoconf automake libtool libpng-devel libjpeg-devel libtiff-devel zlib-devel giflib-devel \
+    && dnf clean all \
+    && rm -rf /var/cache/dnf
+
+
+# handles os dependancies for: crypto + readpdf + parse-emails + docxpy + sklearn + pandas + ippysocks-py3 + oauthlib
+# unzip + py3ews + taxii2 + pan-os-python + slackv3 + google-api-py3 + boto3py3
+# pyjwt3 + joe-security + slack + office-utils
+# note: the packages that are being removed by dnf remove at the end are packages that are required during the installation of python dependancies
+
+# crypto os dependancies: python3-devel gcc gcc-c++ make wget git rust cargo libffi-devel openssl-devel
+# readpdf os dependancies: poppler poppler-utils
+# parse-emails os dependancies: libffi-devel, python3-devel, wget, git
+# docxpy os dependancies: libxml2-devel, libxslt-devel, python3-devel, wget
+# sklearn os dependancies: python3-devel gcc gcc-c++ make wget git openssh curl ca-certificates openssl less make rsync libpng-devel freetype-devel gcc-gfortran openblas unzip
+# pandas os dependancies: python3-devel gcc gcc-c++ make wget git libstdc++
+# ippysocks-py3 os dependancies: gcc python3-devel
+# oauthlib os dependanices: python3-devel gcc gcc-c++ make wget git
+# unzip os dependanices: python3-devel gcc gcc-c++ make wget git p7zip unrar unrar-free
+# py3ews os dependanices: python3-devel gcc gcc-c++ make wget git libxml2-devel openssl-devel
+# taxii2 os dependanices: python3-devel gcc gcc-c++ make wget git
+# pan-os-python os dependanices: python3-devel gcc gcc-c++ make wget git
+# slackv3 os dependanices: python3-devel gcc gcc-c++ make wget git libffi-devel
+# google-api-py3 os dependanices: python3-devel gcc
+# boto3py3 os dependanices: python3-devel gcc gcc-c++ make wget git
+# pyjwt3 os dependanices: python3-devel gcc gcc-c++ make wget git
+# joe-security os dependancies: python3-devel gcc gcc-c++ make wget git
+# slack os dependancies: python3-devel gcc gcc-c++ make wget git libffi
+# office-utils os dependancies: LibreOffice_Linux_x86-64_rpm java-11-openjdk-headless cairo libSM libX11-xcb
+
+
+# line 7 - since in ubi its 7za --> need to make it to run as 7z like our script
+# lines 9-11 is the configuration for sklearn
+# linex 14-16 - install LibreOffice for the office-utils image
+# lines 21-23 - installing libreoffice for the office-utils image
+RUN dnf install -y --nodocs python3-devel gcc gcc-c++ make wget git poppler poppler-utils \
```

Comment:
Any reason why this is not part of the previous `RUN` command (that started on line 9) ?

Reply:
no particular reason, it just to differentiate between chromium + tesseract which are "special" images and more "complex" images than others. 

The tesseract and chromium require more OS dependancies than "regular" docker images. 

I think splitting "complex" docker images (which require a lot of os dependancies) - will make it easier to read in the future. 

Shoving all in one RUN command would make it probably less readable

Reply:
Each RUN command adds a layer -> Larger image size.

Reply:
I know, but for server it does not matter that much, also, the differences aren't so extreme.

Reply:
does it matter so much that we will have 2 RUN commands instead of one? do you think it would make the image significantly smaller image?

Reply:
You can try.
We can also try and squash layers during the build.

Reply:
we try to do it in the next version, i don't think its urgent for the upcoming version, it would require me to test everything out again to see i didn't break something